<launch>
    <node pkg="tf2_ros" type="static_transform_publisher" name="word_slam" args="0 0 0  0.5, -0.5, 0.5, -0.5  world slam" />

    <node pkg="semantic_reconstruction" type="ros_disinf_slam" name="ros_disinf_slam"  output="screen" >
        <param name="model_path"                value="/home/yu/tools/ht_lt.pt" />
        <param name="calib_path"                value="/home/yu/slam_ws/src/RA-SLAM/configs/zed_native_l515.yaml" />
        <param name="orb_vocab_path"            value="/home/yu/tools/orb_vocab.dbow2" />
        <param name="devid"     type="int"      value="0" />
        <param name="renderer"  type="bool"     value="false" />
        <param name="bbox_xy"   type="double"   value="8.0" />

    </node>
    <!-- <include file = "$(find TSDF2Mesh)/launch/tsdf2mesh.launch"/> -->
</launch>